bindings/nodejs/lib/client/client.ts

```diff
@@ -94,6 +94,23 @@ export class Client {
         return JSON.parse(response).payload;
     }
 
+    /**
+     * Fetch alias/basic/NFT output IDs based on the given query parameters.
+     * Supported query parameters are: "hasNativeTokens", "minNativeTokenCount", "maxNativeTokenCount", "unlockableByAddress", "createdBefore", "createdAfter".
```

Comment:
Is this list a subset of all possible query params? Feels like we shouldn't list them, but rather link to the enum or something. Otherwise we'll forget to update this in the future.

Reply:
Yes it's only a subset, because the others don't apply to all 3 output types

But the enum has all variants, not just the supported ones

Reply:
Maybe we should have a different enum then, if some of the variants do nothing?

Reply:
Would be inconsistent because we can't change the other routes without breaking change, do we still want it for this one then?

Reply:
https://github.com/iotaledger/iota-sdk/issues/1248